backend/penndata/views.py

```diff
@@ -283,6 +283,9 @@ def get(self, request):
                 datetime.time(hour=int(hours), minute=int((hours % 1) * 60))
                 for _, hours in self.open_times.values()
             ]
+            # TODO: either unify naming OR add a field to model to indicate which image to use
```

Comment:
should we have this as db field? didn't do this to begin with bc i know it would involve someone manually filling in on admin page

Reply:
Yea I would personally like the db field approach more + making a script to load it in (something like `load_gsrs.py`). that way we can avoid the manual filling